src/magicgui/_util.py

```diff
@@ -187,6 +187,8 @@ def _safe_isinstance_tuple(obj: object, superclass: object) -> bool:
 
 def safe_issubclass(obj: object, superclass: object) -> bool:
     """Safely check if obj is a subclass of superclass."""
+    if obj == superclass:
```

Comment:
when comparing types, shouldn't we use `is` ?

Reply:
```suggestion
    if obj is superclass:
```
lets check